test/regression/issue/21017/index-fixture.html

```
@@ -0,0 +1,6 @@
+<!doctype html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)